vault/forwarded_writer_oss.go

```diff
@@ -0,0 +1,114 @@
+//go:build !enterprise
+
+package vault
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/vault/sdk/logical"
+)
+
+const clusterSentinel = `{{clusterId}}`
```

Comment:
Maybe this constant could be moved into `vault/mount_util_shared.go`, it wouldn't be great if they got out of sync somehow across OSS/ENT.

Reply:
Ah, that might be a good place. I was trying to think of where I wanted this, I was debating somewhere in `sdk/` since this could be used by plugins as well, but I didn't really see a good place... Thoughts?

Reply:
Maybe `sdk/logical` would make the most sense, seeing as this is used with `logical.Storage` from plugin's PoV. I'll update.